gnocchi/metric/v1/measures/results.go

```diff
@@ -0,0 +1,84 @@
+package measures
+
+import (
+	"encoding/json"
+
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// Measure is an datapoint thats is composed with a timestamp and a value.
+type Measure struct {
+	// TimeStamp represents a timestamp of when measure was pushed into the Gnocchi.
+	TimeStamp string `json:"-"`
+
+	// Granularity is a level of precision that is kept when aggregating data.
+	Granularity float64 `json:"-"`
+
+	// Value represents a value of data that was pushed into the Gnocchi.
+	Value float64 `json:"-"`
+}
+
+/*
+UnmarshalJSON helps to unmarshal response from reading Gnocchi measures.
+
+Gnocchi APIv1 returns measures in a such format:
+
+[
+    [
+        "2017-01-08T10:00:00+00:00",
+        300.0,
+        146.0
+    ],
+    [
+        "2017-01-08T10:05:00+00:00",
+        300.0,
+        58.0
+    ]
+]
+
+Helper unmarshals every nested array into the Measure type.
+*/
+func (r *Measure) UnmarshalJSON(b []byte) error {
```

Comment:
This looks correct to me and this is what I would have done. Let's see if @jrperritt knows of an alternative way of doing this.

Reply:
> This looks correct to me

I agree.

A few suggestions:
1. Check the length of the slice is 3 before trying to process it, otherwise it could `panic` (and if it isn't 3, we aren't prepared to process it anyway)
2. `TimeStamp` should be a `time.Time`; you can define a format to parse. See here: https://tip.golang.org/pkg/time/#Parse